modify-variable.md

```diff
@@ -0,0 +1,43 @@
+# `@ModifyVariable`, cancellable
+
+[<- Return](README.md)
+
+Modifies a local variable
+
+Parameters: the type of the local variable (and optionally the arguments of the transformed method)
+
+Return type: the type of the local variable
+
+Example Mixin:
```

Comment:
I'd add an "how to set up correctly" section at the bottom. This section (should maybe be a paragraph actually, in all the pages?) should only have an example mixin, and the equivalent patch for the target method.